### PR TITLE
[SDPA-5217] Ordering lists

### PR DESCRIPTION
**Pull request by yeniatencio**
## Motivation and Context

**JIRA issue:** https://digital-engagement.atlassian.net/browse/SDPA-5217

## Changed

1. Adding padding bottom for after each line
2. Padding top for first child.
3. adding ol and li 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ X] Improvement/refactoring (non-breaking change that doesn't add any features but makes things better)

## Checklist

- [ ] I've added relevant changes to the documentation.
- [ ] I have added tests to cover my changes (if not applicable, please state why)
- [ ] My change requires a template update for create-ripple-app.
- [ ] I have added template update script for next release.

### CODE REVIEW

File: packages/components/Organisms/Markup/Markup.vue [hunk 1]
```diff
@@ -101,6 +101,8 @@ $callout-wrapper-list-padding-left: ($rpl-space * 6) !default;
 $callout-wrapper-padding: ($rpl-space-3) ($rpl-space * 6) ($rpl-space * 6) ($rpl-space * 6) !default;
 $callout-wrapper-padding-rtl: ($rpl-space * 6) ($rpl-space * 6) ($rpl-space-3) ($rpl-space * 6) !default;
 $callout-wrapper-heading-margin: ($rpl-space * 5) 0 !default;
+$list-line-height: $rpl-space * 5 !default;
```
Comment (mayurngondhkar) [inline packages/components/Organisms/Markup/Markup.vue:104]: Hey Yeni,

Suggested variable names

$unordered-list-line-height: $rpl-space * 5 !default;
$unordered-list-padding-bottom: rem(11px) !default;

Thanks

File: packages/components/Organisms/Markup/Markup.vue [hunk 2]
```diff
@@ -119,6 +121,22 @@ $callout-wrapper-heading-margin: ($rpl-space * 5) 0 !default;
     }
   }
 
+  ul li,
+  ol li {
+    line-height: $list-line-height;
+    padding-bottom: $padding-for-li;
+    &:last-child {
+      padding-bottom: 0;
```
Comment (mayurngondhkar) [inline packages/components/Organisms/Markup/Markup.vue:129]: Hey,

Could this use a variable?

**Review by mayurngondhkar — CHANGES_REQUESTED** [review]

File: packages/components/Organisms/Markup/Markup.vue (hunk 1 above)
Comment (yeniatencio) [reply packages/components/Organisms/Markup/Markup.vue]: Done @mayurngondhkar

File: packages/components/Organisms/Markup/Markup.vue (hunk 2 above)
Comment (yeniatencio) [reply packages/components/Organisms/Markup/Markup.vue]: Done. Thanks

**Review by mayurngondhkar — APPROVED** [review]
LGTM

**Pull request merged**